src/rebuild/srv.c

```diff
@@ -908,8 +898,9 @@ rebuild_prepare(struct ds_pool *pool, uint32_t rebuild_ver,
 		ret = rebuild_global_pool_tracker_create(pool, rebuild_ver, rebuild_gen,
 							leader_term, reclaim_eph, rebuild_op, rgt);
 		if (ret) {
-			D_ERROR("rebuild_global_pool_tracker create failed: "DF_RC"\n",
-				DP_RC(ret));
+			DL_ERROR(rc, DF_RB " rebuild_global_pool_tracker create failed",
```

Comment:
a mistake here? Should the DL_ERROR(rc, ...) be placed after the assignment rc = ret?

or DL_ERROR(ret, ...) here.

Note for the next push, which I didn't think would be needed. But maybe due to this issue I'll need to merge with latest master and push for testing.

Reply:
Instead, I'll make the fix in the part 2 patch https://github.com/daos-stack/daos/pull/14547